### ISSUE TITLE
Automatically turn on dark mode with prefers-color-scheme

### ISSUE THREAD

**Issue by Steffan153** [max-programming/max-todos#43]
https://css-tricks.com/dark-modes-with-css/ and https://flaviocopes.com/javascript-detect-dark-mode/ will be useful

**Comment by max-programming** [comment]
Yeah I know that. But I am using Dark Reader. We can use Media Queries by Material UI and it is a Media Query I guess

**Comment by Steffan153** [comment]
I know you use dark reader, but you have a button to turn it on and off so just automatically turn it on when detected

**Comment by max-programming** [comment]
Dark Reader has a method that detects and does something. 
https://github.com/darkreader/darkreader#using-for-a-website
Here, `Darkreader.auto()` is a method that relates to this issue.

**Comment by Steffan153** [comment]
That probably does the same thing, sure.

**Comment by max-programming** [comment]
> That probably does the same thing, sure.
> Edit: Yep https://github.com/darkreader/darkreader/blob/master/src/utils/media-query.ts

But, maybe it didn't seem to work on mine. Try it out on your side, does it work?

**Comment by PriyalPatney** [comment]
How about auto dark mode can use the current time and switch accordingly! 🤔

**Comment by max-programming** [comment]
> How about auto dark mode can use the current time and switch accordingly! 🤔

Maybe we could add a different setting for that. Which can enable dark mode automatically if it's night
